### ISSUE TITLE
Improvement on PHPMailerLangTest.php

### ISSUE THREAD

**Issue by shoko93** [PHPMailer/PHPMailer#1637]
There is a paragraph under "Localization" in the README about PHPMailerLangTest.php as follows.

"We welcome corrections and new languages - if you're looking for corrections to do, run the PHPMailerLangTest.php script in the tests folder and it will show any missing translations."

When I tried to run PHPMaiilerLangTest.php on my computer, I faced some issues so I would like to share those on here.

Firstly, it took too long for `composer install` to install the dependency so I had to reduce the dependency to only PHPUnit in order to run the test. Secondly, I had to change the directory for DirectoryIterator to the following:
```
$path = __DIR__ . '/../language'
foreach (new \DirectoryIterator($path) as $fileInfo)
```
Maybe you could update the file or add note about these on the README?

**Comment by Synchro** [comment]
I can't reasonably cut down the require-dev dependency list because composer only provides a single context to cover all dev requirements - I can't make one specifically for the language test.

Do you want to make a PR for the path change? I would do it as:

    foreach (new \DirectoryIterator(__DIR__ . '/../language') as $fileInfo)

the path is never needed again, so there's no need to save it in a variable.

**Comment by shoko93** [comment]
I understand about the composer, but would it be possible to say something like "Please consider modifying the composer file if it is taking too long to install dependencies." on the README? I believe this is not only for the language test, but it applies to any other part of the development as well. I wanted to contribute, but had trouble with installing dependencies and wasn't really sure what was going on at first. So I think adding such suggestion will give some direction on what to do for people who are willing to help with the development. 

I will make the PR about the path change.

**Comment by Synchro** [comment]
This was released as part of 6.0.7. Thanks for reporting it.
